### ISSUE TITLE
Not changing package name SplashActivity.java

### ISSUE THREAD

**Issue by strdr4605** [junedomingo/react-native-rename#95]
Beside MainActivity and MainApplication I have SplashActivity
![image](https://user-images.githubusercontent.com/16056918/69432367-42539d00-0d42-11ea-8f2e-766febb782fa.png)
When running  

```bash
react-native-rename "Travel App" -b com.junedomingo.travelapp
```
the package name is not replaced in SplashActivity.
Probable because of https://github.com/junedomingo/react-native-rename/blob/master/src/config/bundleIdentifiers.js#L19

**Comment by strdr4605** [comment]
Solved in fork https://github.com/strdr4605/react-native-rename

**Comment by wilsolutions** [comment]
There is a significant number of PRs, I'm wondering if the current maintainer still maintain this repo...

**Comment by vomchik** [comment]
Fixed in the last release v2.7.0
